README.md

```diff
@@ -44,3 +44,9 @@
 
 - The player's password is starred out.
 - The buttons allow the user to edit their profile data and sign out.
+
```

Comment:
Can you move this to the stories as a description for the component? 

I understand that the ticket states to add it onto the readme, this is just for features. Components have the description on their stories file.

Reply:
Have a look at the other stories and you can see how to add a description